osquery/events/events.cpp

```diff
@@ -906,8 +906,8 @@ void attachEvents() {
   for (const auto& subscriber : subscribers) {
     auto status = EventFactory::registerEventSubscriber(subscriber.second);
     if (!status.ok()) {
-      LOG(INFO) << "Error registering subscriber: " << subscriber.first << ": "
-                << status.getMessage();
+      VLOG(1) << "Error registering subscriber: " << subscriber.first << ": "
```

Comment:
My original intention was for the error preventing the loop from happening to have greater prominence than errors for individual subscribers, hence `LOG(WARNING)` above and `VLOG(1)` here. This would also avoid spaming the Error Log if multiple subscribers fail to register.

I also agree that having both be LOG(WARNING) for consistency is another valid approach, and I guess it comes down to what aspects should be prioritized.